jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/awsrules/utils/PacmanUtils.java

```diff
@@ -4162,6 +4181,30 @@ public static String getQualysVulnerabilitiesDetails(JsonArray vulnerabilities)
 
     }
 
+    public static String getCrowdstrikeVulnerabilitiesDetails(JsonArray vulnerabilities) {

```

Comment:
This needs to be fed by a CQ plugin, we cannot expose plugin related login in CE repo and in Core.

This might need to follow the same example as R7 mappings

Reply:
Refactored now so that region is populated in cq plugin and vulnerability url is built in mapper